tailwind-showcases/src/TailwindShowcases.js

```diff
@@ -3,6 +3,7 @@ import { extractClassSuffixes } from './theme-helpers';
 import { getZIndexHtml } from './z-index-helper';
 import { getScaleHtml } from './space-helper';
 import resolveConfig from 'tailwindcss/resolveConfig';
+import 'twind/shim';
```

Comment:
this might override the custom defined tokens from the theme, I think it should be:

![image](https://user-images.githubusercontent.com/7051248/133443033-29538bed-33bd-4d11-b7ea-b932a984f318.png)


and for the docs and stories only - not in the component itself
@gqio @georges-gomes  please advise, because I'm not sure